### PR TITLE
Add behavior param

### PR DESCRIPTION
**Pull request by edwardrf**
Needed for https://github.com/DefangLabs/s.defang.io/pull/2

### CODE REVIEW

File: .github/workflows/test.yaml [hunk 1]
```diff
@@ -24,11 +24,13 @@ jobs:
 
     - name: Deploy
       uses: ./
+      continue-on-error: true # Ignore dry run error
```
Comment (edwardrf) [inline .github/workflows/test.yaml:27]: Ignore error for now, I think the CLI should return 0 on --dry-run if there is no error.
See: https://github.com/DefangLabs/defang/pull/628

File: action.yaml [hunk 2]
```diff
@@ -53,9 +57,13 @@ runs:
             params+=("-f")
             params+=("$filename")
         done
+        if [[ -n ${{ inputs['behavior'] }} ]]; then
```
Comment (lionello) [inline action.yaml:60]: quotes? this fails if the `${{ … }}` is empty

Comment (edwardrf) [reply action.yaml]: Good call, fixed. Added test.

File: .github/workflows/test.yaml [hunk 3]
```diff
@@ -24,14 +24,28 @@ jobs:
...
```
Comment (lionello) [inline .github/workflows/test.yaml:27]: I don't want to merge this

**Review by lionello — CHANGES_REQUESTED** [review]
ignoring errors will affect all our 1-click deploys

File: .github/workflows/test.yaml (hunk 3 above)
Comment (lionello) [reply .github/workflows/test.yaml:27]: for now, better to leave the `defang compose config` and avoid adding the `--behavior` param

**Pull request merged**